docroot/themes/custom/twentyeighteen/sass/components/403-404/_403-404.scss

```diff
@@ -0,0 +1,51 @@
+// 403 and 404 Page Styling
+//
+// Styling 403 and 404 pages
+//
+// Markup: 403-404.twig
+//
+// Style guide: components.403-404
+
+.error-container {
+  position: relative;
+}
+
+.body-sidebars-none .error-container section#content .error-notice {
+  padding: 0;
+}
+
+.error-container {
+  padding-bottom: 10em;
+  background-color: rgba(221, 198, 102, 1);
+
+  @include breakpoint($small-max) {
+    padding-bottom: 0;
+  }
+}
+
+.notfound-backing {
+  background-color: rgba(52, 109, 117, 1);
+}
+
+.error-container #content-area {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 35%;
+  margin: auto;
+  width: 80%;
+  background: rgba(255, 255, 255, 0.95);
+  text-align: center;
+  padding: 2em 1em;
+  box-shadow: 0 0 1em rgba(0, 0, 0, 0.5);
```

Comment:
Same goes as above comment: `box-shadow: 0 0 1em rgba($blk, 0.5);` should work.

Reply:
Done, as with the $wht change above, no discernible difference.